### PR TITLE
Add crd.nsx.vmware.com apiVersion for vpc controllers

### PR DESCRIPTION
**Pull request by zhengxiexie**

### CODE REVIEW

File: build/yaml/crd/t1/nsx.vmware.com_securitypolicies.yaml
Comment (timdengyun) [inline build/yaml/crd/t1/nsx.vmware.com_securitypolicies.yaml:1]: For T1 mode, we should only have securitypolicy and serviceaccount, other VPC related CRDs are not necessary. 
For serviceaccount, I'm not sure, it's good double confirm with @gran-vmv

Comment (gran-vmv) [reply build/yaml/crd/t1/nsx.vmware.com_securitypolicies.yaml:1]: Which thing do you want to confirm?

For NSXServiceAccount, it should work on T1 and VPC mode.

Comment (timdengyun) [reply build/yaml/crd/t1/nsx.vmware.com_securitypolicies.yaml:1]: So, we should keep one T1 mode NSXServiceAccount CRD in group nsx.vmware.com, is it?

Comment (gran-vmv) [reply build/yaml/crd/t1/nsx.vmware.com_securitypolicies.yaml:1]: Yes.

Comment (zhengxiexie) [reply build/yaml/crd/t1/nsx.vmware.com_securitypolicies.yaml:1]: ok

File: pkg/apis/crd.nsx.vmware.com/v1alpha1/nsxserviceaccount_types.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
```
Comment (dantingl) [inline pkg/apis/crd.nsx.vmware.com/v1alpha1/nsxserviceaccount_types.go:1]: Do not change nsxserviceaccount.

Comment (zhengxiexie) [reply pkg/apis/crd.nsx.vmware.com/v1alpha1/nsxserviceaccount_types.go]: ok

**Comment by timdengyun** [comment]
I think you might also change makefile to generate code, we  also checkin it.

**Comment by zhengxiexie** [comment]
> I think you might also change makefile to generate code, we also checkin it.

when `make generated`
```	github.com/vmware-tanzu/nsx-operator/pkg/apis/crd.nsx.vmware.com/v1alpha1: module github.com/vmware-tanzu/nsx-operator/pkg/apis@latest found (v0.1.0), but does not contain package github.com/vmware-tanzu/nsx-operator/pkg/apis/crd.nsx.vmware.com/v1alpha1```
perhaps we should let this PR first check in.

**Comment by zhengxiexie** [comment]
e2e

**Review by dantingl — APPROVED** [review]

**Review by TaoZou1 — APPROVED** [review]

**Pull request merged**